lambda-buffers-proto-compat/src/LambdaBuffers/ProtoCompat/ProtoCompat.hs

```diff
@@ -0,0 +1,562 @@
+{-# LANGUAGE KindSignatures, DuplicateRecordFields #-}
+{-# LANGUAGE DeriveGeneric, TypeSynonymInstances #-}
+{-# LANGUAGE DataKinds, OverloadedLabels #-}
+{-# OPTIONS_GHC -Wno-missing-kind-signatures #-}
+{-# OPTIONS_GHC -Wno-missing-deriving-strategies #-}
+{-# OPTIONS_GHC -Wno-redundant-constraints #-}
+module LambdaBuffers.ProtoCompat.ProtoCompat where
```

Comment:
Explicit exports please, this is not allowed.

Reply:
Er, sorry about that. Everything is cleaned up in #24.